### DISCUSSION TITLE
Rename response

### DISCUSSION THREAD

**Discussion by darkmoonbzh** [mockoon/mockoon#742] [Help] [answered]
Hi,

Is it possible to rename response?
Actually, I have :
 Response 1
 Response 2
 Response 3 etc...

Thanks

**Comment by 255kb** [comment, marked as answer]
Hi, 
You cannot rename the "Response x" part, but you can add some documentation/details using the label input below: 
![image](https://user-images.githubusercontent.com/7489814/176833264-4a3f78ba-14b8-41f9-8604-cf3906aa3486.png)
